### PR TITLE
Add Projector interface

### PR DESCRIPTION
**Pull request by fkarakas**
With this change, it will be possible to provide a projection implementation to allow the projection of messages to an external system (eg redis)

### CODE REVIEW

File: examples/custom_projector/main.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/etf1/kafka-transformer/pkg/transformer/kafka"
+	confluent "gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
+)
+
+// Start a redis with docker : docker run -p 6379:6379 --rm -d redis
```
Comment (iotafr) [inline examples/custom_projector/main.go]: Maybe you can add a redis instance in the docker-compose file ?

Comment (fkarakas) [reply examples/custom_projector/main.go]: The docker-compose is for the kafka-transformer project. kafka-trasnformer is not using a redis projector. That is why i didn't add a redis server in the Docker-compose.yml.

The examples directory is a sub gomodule, totally independent. But if you want i can add a docker-compose.yml in this sub project ?

Comment (iotafr) [reply examples/custom_projector/main.go]: OK for me to add a docker-compose file in this sub projet.

Comment (fkarakas) [reply examples/custom_projector/main.go]: Done

**Pull request merged**